src/main/java/com/team766/robot/constants/ExampleInputConstants.java

```diff
@@ -0,0 +1,39 @@
+package com.team766.robot.constants;
```

Comment:
My understanding of the intent behind `ExampleInputConstants` is that the whole file would be copied and then customized to the specific robot. If we believe that there should be an InputConstants class, I think it would be better to provide that class already with the correct name, but just with example contents that should be replaced.

Let me know if I've interpreted this incorrectly.